wandb/data_types.py

```diff
@@ -436,22 +436,28 @@ def transform(histogram):
 class Table(object):
     MAX_ROWS = 300
 
-    def __init__(self, columns=["Input", "Output", "Expected"], rows=[]):
+    def __init__(self, columns=["Input", "Output", "Expected"], data=[], rows=[]):
```

Comment:
Think you could switch the `[]` in default args to `None`? If you put a list in there and it gets changed, future calls of the function will have the modified list as their default value. Definitely not what you want.

Reply:
Yeah agreed with Adrian, this is a python anti-pattern